AlamofireSessionRenewer/Classes/AlamofireSessionRenewer.swift

```diff
@@ -70,7 +70,7 @@ open class AlamofireSessionRenewer: RequestRetrier {
     // MARK: - RequestRetrier protocol implementation
     
     public func should(_ manager: SessionManager, retry request: Request, with error: Error, completion: @escaping RequestRetryCompletion) {
-        if (error as NSError).domain == errorDomain, (error as NSError).code == authenticationErrorCode {
+        if credential != nil, (error as NSError).domain == errorDomain, (error as NSError).code == authenticationErrorCode {
```

Comment:
@yakovshkolnikov May we create open method in AlamofireSessionRenewer which validates credential so clients may provide their own way to validate credential?

Reply:
We already have: open func isCredentialEqual(to request: Request) -> Bool

Reply:
So why are we using credential != nil check here?

Reply:
Lets create separate method isCredentialEmpty so logic in should retry method will be more readable